modules/core/04-channel/keeper/grpc_query_test.go

```diff
@@ -11,6 +11,7 @@ import (
 	"github.com/cosmos/ibc-go/v7/modules/core/04-channel/types"
 	"github.com/cosmos/ibc-go/v7/modules/core/exported"
 	ibctesting "github.com/cosmos/ibc-go/v7/testing"
+	"github.com/cosmos/ibc-go/v7/testing/mock"
 )
 
 const doesnotexist = "doesnotexist"
```

Comment:
this is unrelated to this PR i guess, but this const is a bit random here if we are just using string values in all the rest of the tests